### ISSUE TITLE
Idea: exclude cells from printing

### ISSUE THREAD

**Issue by mojodna** [fieldpapers/fieldpapers#235]
When producing an atlas for an area, it occasionally overlaps w/ areas that are not intended to be enumerated. As such, it's a waste of paper (or PDF space) to create pages for them.

Potential UI: select a grid cell (/ atlas page) and select an option that excludes it from being printed.

**Comment by aawiseman** [comment]
Agreed, this would be helpful when you are only doing a specific area and don't need every square.

**Comment by the-knife** [comment]
Agreed.
